### PR TITLE
Filename from docuemnt url

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-272

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java [hunk 1]
```diff
@@ -64,4 +64,16 @@ private static Document mapScannedDocument(ScannedDocument doc) {
...
+
+    private static String getFileName(ScannedDocument doc) {
+        if (doc.fileName == null) {
+            if (doc.url == null) {
+                return null;
+            } else {
+                return doc.url.documentFilename;
+            }
+        } else {
+            return doc.fileName;
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java:76]: are we sure doc.fileName == doc.url.documentFilename. ??

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java:76]: No. But this does not matter for us - we are checking if not both are null

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ private static Document mapScannedDocument(ScannedDocument doc) {
             return null;
         } else {
             return new Document(
-                doc.fileName,
+                getFileName(doc),
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java:57]: here you are using 'doc.url.documentFilename' if it is not null

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java:57]: Not exactly. I am using _doc.url.documentFilename_ if _doc.url_ is not null

**Comment by apereverzin** [comment]
Not necessary

**Pull request closed without merging**